packages/manager/src/features/CloudPulse/Utils/models.ts

```diff
@@ -185,3 +185,18 @@ export enum CloudPulseSelectTypes {
   // eslint-disable-next-line sonarjs/future-reserved-words
   static,
 }
+
+/**
+ * CloudPulseAvailableViews holds the views in which the filter should be shown
+ */
+export enum CloudPulseAvailableViews {
+  /**
+   * This is the centralized view, where the filter should be shown
+   */
```

Comment:
should this be a bit more specific? eg
```suggestion
  /**
   * Represents the centralized Metrics view (at /metrics). If present in a filter's `neededInViews` configuration, that filter will be shown in the central view
   */
```